core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2369,6 +2370,16 @@ private void loadFileMetadataAndJournal(LockedInodePath inodePath,
       createFileAndJournal(inodePath, createFileOptions, journalContext);
       CompleteFileOptions completeOptions = CompleteFileOptions.defaults().setUfsLength(ufsLength);
       completeFileAndJournal(inodePath, completeOptions, journalContext);
+      if (inodePath.getLockMode() == LockMode.READ) {
```

Comment:
This can be `InodeTree.LockMode.READ` and the import doesn't have to change. This is consistent with other usage in this file.

Changing all the other usage can be done in a separate PR.